authority/policy.go

```diff
@@ -0,0 +1,405 @@
+package authority
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"go.step.sm/linkedca"
+
+	"github.com/smallstep/certificates/authority/admin"
+	authPolicy "github.com/smallstep/certificates/authority/policy"
+	policy "github.com/smallstep/certificates/policy"
+)
+
+type policyErrorType int
+
+const (
+	_ policyErrorType = iota
+	AdminLockOut
```

Comment:
```suggestion
	AdminLockOut policyErrorType = iota + 1
```

Reply:
Fixed in 2a7620641f74474a7a055e20c779e6c70293c32d.